src/charge_controller/ChargeController.cpp

```diff
@@ -149,3 +190,60 @@ void ChargeController::stopCharging() {
         changedState = true;
     }
 }
+
+void ChargeController::sendChargerMessage() {
+    uint16_t voltage = 0;
+    uint16_t current = 0;
+    uint8_t shouldCharge = 1;
+
+    if (state == ControllerStates::CHARGING) {
+        // Multiply by ten to get the right sized values
+        voltage = 48 * 10;
+        current = 60 * 10;
+        shouldCharge = 0;
+    }
```

Comment:
Check whether should shouldCharge is 0 or 1 to enable.

Reply:
I looked into this a little more to make sure that these were indeed the right values. After re-finding the [CAN Specification](https://github.com/RIT-EVT/charge-controller/blob/charge-controller-bringup/datasheets/Elcon%20CAN%20Specification.pdf) it confirms that 1 is Stop Charging, and 0 is Start charging. I think changing the variable name here to stopCharge would be a better variable name